tests/integration_tests/validation/conformance_suite_configurations/edinet.py

```diff
@@ -120,6 +119,9 @@
     "EC5700W.GFM.1.8.3/index.xml:invalid01": "valid/index.xml:valid05",
     "EC5700W.GFM.1.9.1/index.xml:invalid01": "valid/index.xml:valid05",
     "EC5700W.GFM.1.10.3/index.xml:invalid01": "valid/index.xml:valid05",
+    "EC5700W.GFM.2.5.1/index.xml:invalid01": "valid/index.xml:valid05",
+    "EC5700W.GFM.2.6.1/index.xml:invalid01": "valid/index.xml:valid05",
+    "EC5700W.GFM.2.8.1/index.xml:invalid01": "valid/index.xml:valid05",
```

Comment:
No need to change for this one, but just FYI that `valid09` runs something like 10 times faster than `valid05`, so I always try to build testcases from that first (sometimes it's not possible, in which case others are fine).

Here's the samples sorted by size, which roughly correlates to processing time:
<img width="249" height="347" alt="Image" src="https://github.com/user-attachments/assets/2aab5d7e-0fa0-409d-bfad-f8b505d8034d" />

Reply:
Shoot, I thought I was working with the fastest. Ill make note of that and use 9 moving forward.